packages/webapp/webapp_tests.js

```diff
@@ -1,13 +1,15 @@
 var url = Npm.require("url");
```

Comment:
Let's just switch these tests to depending on `ecmascript` so we can use `import`s (or at least `require`) here.